pkg/cli/images.go

```diff
@@ -103,26 +144,29 @@ func printContainerImages(images []apiv1.Image, ctx context.Context, c client.Cl
 	}
 
 	for _, image := range images {
-		if image.Tag == "" && image.Repository == "" {
-			if !a.All {
-				continue
-			}
-			if !a.Quiet {
-				image.Tag = "<none>"
-				image.Repository = "<none>"
-			} else {
-				image.Tag = image.Name
-				image.Repository = "<none>"
-			}
-		}
-
 		containerImages, err := getImageContainers(c, ctx, image)
 		if err != nil {
 			return err
 		}
 
 		for _, imgContainer := range containerImages {
-			out.Write(imgContainer)
+			imageContainerPrint := imageContainerPrint{
```

Comment:
Hm. No, it's still shadowed. The name of the variable is imageContainerPrint, which is also the name of the struct.

There's several other instances of this throughout the PR as well. As a general rule, i'd like you to avoid this.

Reply:
hmm it should be changed maybe looking at old file?